core/services/workflows/metering/metering.go

```diff
@@ -378,6 +388,13 @@ func (r *Report) Settle(ref string, spendsByNode []capabilities.MeteringNodeDeta
 				continue
 			}
 
+			if isGasSpendType(unit) {
+				// TODO: this decimal shift should be temporary and converted when write capabilities
```

Comment:
Do we have a ticket tracking this?

Reply:
If we haven't completed the existing ticket for adding spend to capabilities, then this note should probably go away and we just assume `big.Int`. Then make a note in the existing ticket about ensuring spend is reported as `big.Int`.